portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,28 @@
 #greeting-container {
   margin-top: 20px;
 }
+body{
+    padding: 0;
+    margin: 0;
+    position: relative;
+}
+span{
+    position: absolute;
+    width: 100vw;
+    background: rgb(245, 102, 102);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 1rem;
+}
+span > *{
+    flex: 1;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    color: white;
+    font-size: 1rem;
+    /* text-transform: uppercase; */
+}
+img{
+    width: 50vw;
+    min-width: 400px;
+}
```

Comment:
Couple formatting violations:
- "Indent by two spaces at a time." (might wanna configure your editor to use 2 space indentation) - https://google.github.io/styleguide/htmlcssguide.html#Indentation
- "Use a space between the last selector and the declaration block." - https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation
- "Separate rules by new lines." - https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation